test/javascripts/acceptance/admin-user-emails-test.js.es6

```diff
@@ -0,0 +1,129 @@
+import { acceptance } from "helpers/qunit-helpers";
+
+acceptance("Admin - User Emails", { loggedIn: true });
+
+const response_with_secondary = secondary_emails => {
+  return [200, { "Content-Type": "application/json" }, {
+    id: 1,
+    username: "eviltrout",
+    email: "eviltrout@example.com",
+    secondary_emails: secondary_emails
+  }];
+};
+
+const assert_no_secondary = assert => {
+  assert.equal(
+    find(".display-row.email .value a").text(),
+    "eviltrout@example.com",
+    "displays primary email"
+  );
+  assert.equal(
+    find(".display-row.secondary-emails .value").text().trim(),
+    "No secondary emails",
+    "displays no secondary emails"
+  );
+};
+
+QUnit.test("my account has no secondary emails", assert => {
+  server.get("/admin/users/1.json", () => { // eslint-disable-line no-undef
+    return response_with_secondary([]);
+  });
+  visit("/admin/users/1/eviltrout");
+
+  andThen(() => {
+    assert_no_secondary(assert);
+  });
+});
+
+QUnit.test("my account has one secondary email", assert => {
+  server.get("/admin/users/1.json", () => { // eslint-disable-line no-undef
+    return response_with_secondary(["eviltrout1@example.com"]);
+  });
+  visit("/admin/users/1/eviltrout");
+
+  andThen(() => {
+    assert.equal(
+      find(".display-row.email .value a").text(),
+      "eviltrout@example.com",
+      "displays primary email"
+    );
+    assert.equal(
+      find(".display-row.secondary-emails .value a").text(),
+      "eviltrout1@example.com",
+      "displays secondary email"
+    );
+  });
+});
+
+const assert_multiple_secondary = assert => {
+  assert.equal(
+    find(".display-row.email .value a").text(),
+    "eviltrout@example.com",
+    "displays primary email"
```

Comment:
I don't think it is necessary to exert for the primary email here.

Reply:
That's fair, the number of secondary emails shouldn't affect the rendering of the primary email - which we've already tested for.